public/assets/sass/components/_navigation.scss

```diff
@@ -0,0 +1,12 @@
+.navigation {
+    // display: flex;
+    // justify-content: space-between;
+    // align-items: center;
+    margin: 7px;
+    font-size: 1.3rem;
+    border-bottom: 1px solid $gray-lighter;
+
+//     @include respond(navbar){
+
+//     }
```

Comment:
Kindly, remove unneeded snippets. If they are saved for future implementation, then add a TODO comment above the statement.